stores/metadata.go

```diff
@@ -576,6 +583,59 @@ func (s *SQLStore) MarkPackedSlabsUploaded(ctx context.Context, slabs []api.Uplo
 	return nil
 }
 
+func (s *SQLStore) pruneHostSectorLoop() {
+	ticker := time.NewTicker(5 * time.Minute)
```

Comment:
I was on the fence whether to add it or not... I still am, figured to propose it but fine to remove it.

Reply:
I don't think these queries would acquire the same locks. I ended up removing the ticker, I think it was a mistake to add it in the first place. I added it because I wanted to ensure the pruning occurred with some frequency in case we forget to trigger the loop on a contract archival event. That would hide the issue for a long time though so it is better to remove this false sense of safety and have it be more in our faces in case it does happen.

Reply:
I personally still think it's better to have two loops here. Simply because we have to trigger it and it makes little sense to trigger the one loop and every event that triggers the other and vice versa...